src/Microsoft.Azure.Relay/HybridConnectionClient.cs

```diff
@@ -139,6 +139,16 @@ public HybridConnectionClient(string connectionString, string path)
         [EditorBrowsable(EditorBrowsableState.Never)]
         public bool UseBuiltInClientWebSocket { get; set; }
 
+        /// <summary>
+        /// Websocket's keep-alive interval.
+        /// </summary>
+        public TimeSpan KeepAliveInterval { get; set; }
+
+        /// <summary>
+        /// Custom ClientWebSocketFactory Implementation.
+        /// </summary>
+        public IClientWebSocketFactory CustomClientWebSocketFactory { get; set; }
```

Comment:
>CustomClientWebSocketFactory [](start = 39, length = 28)

Can we consider applying `[EditorBrowsable(EditorBrowsableState.Never)]` on here? It's an advanced scenario.

Reply:
added